oojs.js

```diff
@@ -1,12 +1,33 @@
 // Define a Playlist class. It should have...
+//
   // A title property that is determined by some input.
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
+  music class Playlist {
+    constructor(songTitle, genre){
+    this.songTitle = songTitle;
+    this.genre = genre;
+    this.songs = ["Achey Breaky Heart", "Bad to the Bone", "Miami"]
+    favorites = 0
+  },
+  addSong(addSong) {
+    this.songs.add(addSong);
+  },
 
-// Create an instance of the Playlist class.
 
+  }
+
+// Create an instance of the Playlist class.
+Playlist myPlayList = new Playlist();
```

Comment:
`myPlaylist` is a variable, so it should be preceded by `var`, `let` or `const`. `Playlist` should only be on the right side of this assignment.